source/clear-linux/tutorials/tutorials.rst

```diff
@@ -24,5 +24,7 @@ Explore our tutorials to discover what you can do with |CL|!
    kubernetes/kubernetes*
    yubikey-u2f
    nvidia
+   nvidia-cuda
+   dars
```

Comment:
Remove reference to `dars`, please.  It's been moved here: 
https://github.com/clearlinux/clear-linux-documentation/tree/master/source/clear-linux/guides/stacks